packages/web3-core/CHANGELOG.md

```diff
@@ -99,6 +99,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   If a transaction object with a `data` property is passed to `txInputOptionsFormatter`, it will now be replaced with `input` (#5915)
 -   The types `TransactionTypeParser` and `TransactionBuilder` are now utilizing the type `Transaction` for the transaction object. (#5993)
 -   no need for polyfilling nodejs `net` and `fs` modules
+-   Changed directly emitting events to `_triggerConfigChange` in `Web3Config` class (#5813)
```

Comment:
```suggestion
-   `Web3Config` now consistently uses `_triggerConfigChange` (#5813)
```

Although I don't really think a change to the `CHANGELOG.md` is needed since this is an internal change that won't affect users

Reply:
@Parthib314, 
 Generally only notable changes are mentioned in changelog that effect end users. IMO this PR doesn't effect end uses so either you may remove your updates in changelog files or conflicts in few files needs to be fixed before we merge . 
 
This PR LGTM, Thanks for your contribution in 4.x.